core/src/main/java/nl/nn/adapterframework/parameters/Parameter.java

```diff
@@ -945,7 +945,7 @@ public void setXpathExpression(String xpathExpression) {
 	}
 
 	/**
-	 * when set to <code>2</code> xslt processor 2.0 (net.sf.saxon) will be used, otherwise xslt processor 1.0 (org.apache.xalan). <code>0</code> will auto detect
+	 * when set to <code>2</code> xslt processor 2.0 (net.sf.saxon) will be used, otherwise xslt processor 1.0 (org.apache.xalan). <code>0</code> will auto detect. This class supports up to and including XSLT version 3.0.
```

Comment:
Actually this comment should be updated to say that the Saxon processor will be used when setting this parameter to 2 or 3.